superset-frontend/cypress-base/cypress/integration/explore/visualizations/line.test.ts

```diff
@@ -19,7 +19,11 @@
 import { FORM_DATA_DEFAULTS, NUM_METRIC, SIMPLE_FILTER } from './shared.helper';
 
 describe('Visualization > Line', () => {
-  const LINE_CHART_DEFAULTS = { ...FORM_DATA_DEFAULTS, viz_type: 'line' };
+  const LINE_CHART_DEFAULTS = {
+    ...FORM_DATA_DEFAULTS,
+    viz_type: 'line',
+    show_legend: true,
```

Comment:
Legends were disabled by default in https://github.com/apache-superset/superset-ui/pull/904, but unfortunately the falsy default value still caused the legend to be rendered (I assume `undefined` reverts to the default value in NVD3 which is `true`). This updates the test to explicitly enable the legend to ensure that the `.nv-legend-text` class is found on line 159 and other places here.

Reply:
Before, the "Legend" checkbox would be unchecked, but the legend would still display due to the control value being `undefined`:
![image](https://user-images.githubusercontent.com/33317356/151506955-b7c3c4dc-ff66-431e-b91f-c98882623f60.png)

After this, the legend isn't displayed as the control value is properly set to the default value (`false`):
![image](https://user-images.githubusercontent.com/33317356/151506746-4b191336-5666-4456-a824-3f8c296eabff.png)